### ISSUE TITLE
Profiler tab is missing in React DevTools (Chrome extension)

### ISSUE THREAD

**Issue by maxdonetsk** [facebook/react-devtools#1325]
Hi, could you please suggest why Profiler tab is missing in ReactDevTools?
I have this in ```package.json```:
```
...
"react": "^16.8.3",
...
"react-dom": "^16.8.3",
...
```
But Profiler tab is absent.
What could I do wrong?

**Comment by maxdonetsk** [comment]
Oh, I must add ```React Performance Devtool``` to make ```Profiler``` this tab available! Found this info here: https://scotch.io/tutorials/use-the-react-profiler-for-performance

**Comment by skirankumar7** [comment]
> Oh, I must add `React Performance Devtool` to make `Profiler` this tab available! Found this info here: https://scotch.io/tutorials/use-the-react-profiler-for-performance

I tried this but didn't worked for me.

**I am working on development mode only using webpack dev server in local but I am not getting show up that profiler tab section in react devtools I have "react": "^16.8.6", "react-dom": "^16.6.3",
I have installed react developer tools extension of the latest version.
Help me with this,
I can have even skype call also if someone wants to help me.
Thanks.**
